libs/openchallenges/team/src/lib/team.component.ts

```diff
@@ -52,22 +52,22 @@ export class TeamComponent implements OnInit {
       objectKey: 'openchallenges-icon.svg',
     });
     this.thomas$ = this.imageService.getImage({
-      objectKey: 'team/thomas.png',
+      objectKey: 'team/tschaffter.jpeg',
```

Comment:
Add constraints to the images. The radius of the avatar circle seems to be 203 px (?), so requesting Thumbor to return 250x250 should be enough. However, I suggest to use 500x500 px for higher quality and as a higher-quality asset that someone may be interested in using (e.g. in presentation).

```suggestion
      objectKey: 'team/tschaffter.jpeg',
      height: ImageHeight._500px,
      aspectRatio: ImageAspectRatio._11,
```

Reply:
The radius of the avatar circle has been changed to 240px (was flexible). 
I agree to request larger images sizes and I will make the adjustments soon.